content/01.abstract.md

```diff
@@ -7,3 +7,10 @@ Here we present a collaborative effort to organize and consolidate the rapidly e
 We present information about the virus in the context of what is known about related viruses and synthesize studies emerging about the diagnosis and treatment of COVID-19 alongside literature about related illnesses.
 A broad scientific effort to understand this pandemic and related viruses and diseases will be foundational to efforts to predict possible interventions. 
 This text is an evolving and collaborative document that seeks to incorporate the ever-expanding body of information related to SARS-CoV-2 and COVID-19.
+
+## Where to Contribute
+Introduce Yourself (GitHub Issue) <https://github.com/greenelab/covid19-review/issues/17>
```

Comment:
@mprobson are you okay switching to Markdown link syntax?  It can make it easier for reviewers to use the GitHub rich diff to verify the link works as intended instead of having to use the rendered manuscript from the artifacts.

Reply:
Sure! I was trying it out based on manubot's link to pandoc but I'm used to md links. My comment below was sent before I saw this.